src/riak_kv_wm_timeseries_listkeys.erl

```diff
@@ -165,14 +165,10 @@ ts_keys_to_body(EncodedKeys, Table, Mod) ->
     BaseUrl = base_url(Table),
     Keys = decode_keys(EncodedKeys),
     KeyTypes = riak_kv_wm_ts_util:local_key_fields_and_types(Mod),
-    URLs = [format_url(BaseUrl, KeyTypes, Key)
-            || Key <- Keys],
-    %% Dialyzer does not like the list_comprehension, if you want to avoid the
-    %% dialyzer error you have to write it like this:
-    %% URLs = lists:map(fun(Key) ->
-    %%                          format_url(BaseUrl, KeyTypes, Key)
-    %%                  end,
-    %%                  Keys),
+    URLs = lists:map(fun(Key) ->
```

Comment:
Please leave in a comment on dialyzer, with a `@todo` comment and the code that it does not accept.